torchci/rockset/commons/__sql/slow_tests.sql

```diff
@@ -0,0 +1,74 @@
+WITH most_recent_strict_commits AS (
+    SELECT
+        push.head_commit.id as sha,
+    FROM
+        commons.push
+    WHERE
+        push.ref = 'refs/heads/viable/strict'
+        AND push.repository.full_name = 'pytorch/pytorch'
+    ORDER BY
+        push._event_time DESC
+    LIMIT
+        3
+), workflows AS (
+    SELECT
+        id
+    FROM
+        commons.workflow_run w
+        INNER JOIN most_recent_strict_commits c on w.head_sha = c.sha
+    WHERE
+        w.name != 'periodic'
+),
+job AS (
+    SELECT
+        j.id
+    FROM
+        commons.workflow_job j
+        INNER JOIN workflows w on w.id = j.run_id
+    WHERE
+        j.name NOT LIKE '%asan%'
+),
+duration_per_job AS (
```

Comment:
this query is more duration per class, right?

Reply:
it's the duration of each test on a job, which is then later averaged to create the average test time across all jobs.